yarn.lock

```diff
@@ -0,0 +1,3973 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Was this added accidentally?

Reply:
That's a good question - it was deliberate to switch to using `yarn`, but I should have cleaned up by removing the package-lock.json file. I cleaned that up and switched the prebuild command to use yarn in b677f6d.